docs/rules/no-multiple-actions-in-effects.md

```diff
@@ -24,21 +24,21 @@ export class Effects {
           catchError((error) => loadEmployeeListError(error)),
         ),
       ),
-    );
+    )
```

Comment:
The changes in this file were made after run `prettier` manually, if it isn't desired, I can revert.

Reply:
That's fine 👍
It's probably a commit that didn't run the prettier hook.